### PR TITLE
Implement an EVERYONE principal in WebACAuthorizationDelegate.

### PR DESCRIPTION
**Pull request by peichman-umd**
- Name of the principal is the string value of the URI for foaf:Agent

### CODE REVIEW

File: src/main/java/org/fcrepo/auth/webac/URIConstants.java [hunk 1]
```diff
@@ -109,6 +109,12 @@
     public static final String WEBAC_ACCESS_CONTROL_VALUE = WEBAC_NAMESPACE_VALUE + "accessControl";
     public static final URI WEBAC_ACCESS_CONTROL = URI.create(WEBAC_ACCESS_CONTROL_VALUE);
 
+    /**
+     * FOAF Agent
+     */
+    public static final String FOAF_AGENT_VALUE = "http://xmlns.com/foaf/0.1/Agent";
```
Comment (awoods) [inline src/main/java/org/fcrepo/auth/webac/URIConstants.java]: To make `foaf` more reusable, maybe you can use the pattern established above of defining the `foaf` namespace, then your `FOAF_AGENT_VALUE` can append to the namespace.

**Comment by awoods** [comment]
Looks good, pending response to code review comment: 
https://github.com/fcrepo4-labs/fcrepo-module-auth-webac/pull/11#discussion_r38455464

**Comment by whikloj** [comment]
:+1: pending @awoods review comment.

File: src/main/java/org/fcrepo/auth/webac/URIConstants.java (hunk 1 above)
Comment (peichman-umd) [inline src/main/java/org/fcrepo/auth/webac/URIConstants.java]: That would be simple enough to add, though do we foresee using more FOAF properties or classes in the WebAC auth module? The YAGNI principle would pull in the direction of leaving it as-is until we need to add the next FOAF-namespaced constant.

Comment (ajs6f) [inline src/main/java/org/fcrepo/auth/webac/URIConstants.java]: For my money, YAGNI is really about functionality. This is about code sanitation. A separate namespace is easier to read and understand.

Comment (awoods) [inline src/main/java/org/fcrepo/auth/webac/URIConstants.java]: Additionally, `foaf:member` is expected for the `acl:agentClass` scenario.
https://www.w3.org/wiki/WebAccessControl

Comment (peichman-umd) [inline src/main/java/org/fcrepo/auth/webac/URIConstants.java]: Both good reasons. Stand by for updated PR.

**Comment by acoburn** [comment]
:+1:

**Comment by acoburn** [comment]
Merged via 6a6b0ff293eac5e9a14ae62733db31e7c3204af2

**Pull request closed without merging**